wagtail/contrib/search_promotions/forms.py

```diff
@@ -61,12 +102,26 @@ def add_fields(self, form, *args, **kwargs):
         del form.fields["query"]
 
     def clean(self):
-        # Search pick must have at least one recommended page to be valid
-        # Check there is at least one non-deleted form.
+        """
+        Search pick must have at least one recommended page or link to be valid.
+        Check there is at least one non-deleted form with the required field.
+        """
```

Comment:
This diff would be simpler to read if we only changed the message rather than also updated the comment style.

Reply:
I personally think it makes sense to update this to docstring while changing, this will be fine.